src/tests/ftest/verify_perms.py

```diff
@@ -324,8 +321,8 @@ def _run_as_user():
         '''Set uid/gid before executing the method.'''
         try:
             try:
-                os.setregid(gid, gid)
-                os.setreuid(uid, uid)
+                os.setregid(gid, gid)       # pylint: disable=no-member
+                os.setreuid(uid, uid)       # pylint: disable=no-member
```

Comment:
I'm surprised there was a pylint about this

Reply:
It was windows-specific.  Removed.